src/pudl_rmi/coordinate.py

```diff
@@ -55,7 +56,9 @@ def __init__(self, pudl_out):
             )
 
     # @profile
-    def plant_parts_eia(self, clobber=False):
+    def plant_parts_eia(
+        self, clobber=False, pickle_distinct=False, pickle_train_connections=False
+    ):
```

Comment:
It makes sense for memory efficiency to create the distinct plant parts list and connect the training data to EIA true granularity records while the plant parts list is already loaded (so we don't gotta load it later). These two inputs are needed for the connection from FERC to EIA.

In the CI `test_plant_parts_list` comes before the `test_connect_ferc1_to_eia` so this works well.

Reply:
If I understand correctly, you're saying that it makes more sense organizationally to have all the distinct-ing be done in `plant_parts_eia_distinct` where it would get the PPE and make distinct and `plant_parts_eia` would never have an argument to make it distinct. I agree that including the `pickle_distinct` arg in `plant_parts_eia` is confusing/circular and doesn't isolate the functionality of `plant_parts_eia_distinct`. 

Maybe it's enough to just add a sentence in the docstring for the `pickle_distinct` arg that this is used for memory efficiency when running the CI? It definitely feels a little janky rn but it's also definitely more efficient to create the distinct when the full PPL is created and loaded.

Reply:
I made this less weird. Now there's just a separate output function for `plant_parts_eia_distinct` that grabs the plant part list, makes it distinct, and pickles the output.